peach_tuntiwong/week01/day4/js/main.js

```diff
@@ -1,3 +1,73 @@
+// Bank
+
+var getAccountIndex = function(user, accList) {
+  for (i in accList) {
+    if (user === accList[i].name) {
+      return i;
+    }
+  }
+  return -1;
+}
+
+var bank = {
+    account: [
```

Comment:
`accounts` (plural) might be a better name here, to indicate that you have an array storing multiple accounts, not just one.